OWNERS_ALIASES

```diff
@@ -127,6 +127,10 @@ aliases:
     - gochist
     - ianychoi
     - seokho-son
+  sig-docs-ta-owners: # Admins for Tamil content
```

Comment:
This section needs two contributors at a minimum, so that you have at least two people who can approve each other's PRs.